flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/JsonGenerateUtils.scala

```diff
@@ -31,12 +31,27 @@ import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.node.{Arr
 import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.util.RawValue
 
 import org.apache.calcite.rex.{RexCall, RexNode}
+import org.apache.calcite.sql.SqlJsonConstructorNullClause
 
 import java.time.format.DateTimeFormatter
 
 /** Utility for generating JSON function calls. */
 object JsonGenerateUtils {
 
+  /** Returns a term which wraps the given <code>expression</code> into a [[JsonNode]]. If the
```

Comment:
nit: `<code>` is not Scala

Reply:
These would be so much easier to spot if IntelliJ didn't happily render them correctly…